guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -60,3 +60,45 @@ Create a custom VCL snippet to rewrite the path for `sitemap.xml` to `/media/sit
   "content": "if ( req.url.path ~ \"^/?sitemap.xml$\" ) { set req.url = \"/media/sitemap.xml\"; }"
 }
 ```
+
+See the example below to understand how to rewrite the path for `robots.txt` and `sitemap.xml` to `/media/robots.txt` and `/media/sitemap.xml` accordingly in the single snippet:
+
+```json
+{
+  "name": "sitemaprobots_rewrite",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "90",
+  "content": "if ( req.url.path ~ \"^/?sitemap.xml$\" ) { set req.url = \"/media/sitemap.xml\"; } else if (req.url.path ~ \"^/?robots.txt$\") { set req.url = \"/media/robots.txt\";}"
+}
+```
+
+#### To use a Fastly VCL snippet for particular domain redirect:
+
+Create a `pub/media/domain_robots.txt` file, where the domain is `domain.com` and use the next VCL snippet:
+
+```
+{
+  "name": "domain_robots",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "90",
+  "content": "if ( req.url.path == \"/robots.txt\" ) { if ( req.http.host ~ \"(domain).com$\" ) { set req.url = \"/media/\" re.group.1 \"_robots.txt\"; }}"
+}
+```
+
+VCL will take care of routing `http://domain.com/robots.txt` and present the correct file.
+
+To configure redirect for `robots.txt` and `sitemap.xml` in the single snippet, create `pub/media/domain_robots.txt` and `pub/media/domain_sitemap.xml`, where the domain is `domain.com` and use the next VCL snippet:
+
+```
+{
+  "name": "domain_sitemaprobots",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "90",
+  "content": "if ( req.url.path == \"/robots.txt\" ) { if ( req.http.host ~ \"(domain).com$\" ) { set req.url = \"/media/\" re.group.1 \"_robots.txt\"; }} else if ( req.url.path == \"/sitemap.xml\" ) { if ( req.http.host ~ \"(domain).com$\" ) {  set req.url = \"/media/\" re.group.1 \"_sitemap.xml\"; }}"
+}
+```
+
+The admin config for the `sitemap` you will need to specify that the file actually is in `pub/media/` rather than `/`.
```

Comment:
```suggestion
In the `sitemap` admin config, you must specify the location of the file using `pub/media/` rather than `/`.
```
Might be nice to explain why.

Reply:
**done**